src/Database/Traits/Sluggable.php

```diff
@@ -88,7 +88,13 @@ protected function getSluggableUniqueAttributeValue($name, $value)
         $separator = $this->getSluggableSeparator();
         $_value = $value;
 
-        while ($this->newSluggableQuery()->where($name, $_value)->count() > 0) {
+        if (method_exists($this, 'withTrashed') ) {
```

Comment:
I think you should be using the `$this->methodExists('withTrashed')` method instead because that takes into account dynamic methods.

Reply:
@Adrion ^